app/screens/HomeScreen.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react'
+import {
+  SafeAreaView,
+  View,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  Image,
+  FlatList,
+} from 'react-native'
+import { connect } from 'react-redux'
+import WashTimeCell from '../components/WashTimeCell'
+import { colors } from '../config'
+import logo from '../assets/images/logo.png'
+import ShiftView from '../components/ShiftView'
+import { startShift } from '../state/Shift'
+import { useDispatch } from 'react-redux'
+
+export const HomeScreen = ({ started }) => {
+  const dispatch = useDispatch()
+
+  const startShiftButtonClicked = () => {
+    dispatch(startShift(!started))
+  }
+
+  const keyExtractor = (item, index) => `${index}`
+
+  const renderItem = ({ item, index }) => (
+    <WashTimeCell item={item} index={index} key={`${index} ${item.id}`} />
+  )
+
+  const washButtonClicked = () => {}
+
+  const {
+    mainViewStyle,
+    washButtonBgViewStyle,
+    washButtonViewStyle,
+    historyTextViewStyle,
+    flatListStyle,
+    startShiftButtonStyle,
+    startShiftStyle,
+  } = styles
+  const startShiftText = started ? 'End Shift' : 'Start Shift'
+  const washHistory = [{ id: '1', dateTime: Date.now() }]
```

Comment:
This looks like something that could make sense in redux when we start manipulating the data. Not something that needs to change immediately but worth thinking about how we'll move it there before we start changing the array.

Reply:
History is not saving in redux yet. So for testing purpose I have used local array. so I have create new redux state for the same.